src/routes/index.js

```diff
@@ -22,4 +23,5 @@ router.use('/questions', question)
 router.use('/resources-categories', resourcesCategory)
 router.use('/offers', offer)
 router.use('/locations', locationRoutes)
-router.use('/categories', category)
\ No newline at end of file
+router.use('/categories', category)
+router.use('/subjects', subject)
```

Comment:
Have you thought about such a route?
```suggestion
router.use('categories/:id/subjects', subject)
```